keyboards/keychron/q1/matrix.c

```diff
@@ -0,0 +1,222 @@
+/* Copyright 2022 @ Keychron (https://www.keychron.com)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "matrix.h"
+#include "quantum.h"
+
+// Pin connected to DS of 74HC595
+#define DATA_PIN A7
+// Pin connected to SH_CP of 74HC595
+#define CLOCK_PIN B1
+// Pin connected to ST_CP of 74HC595
+#define LATCH_PIN B0
+
+#ifdef MATRIX_ROW_PINS
+static pin_t row_pins[MATRIX_ROWS] = MATRIX_ROW_PINS;
+#endif // MATRIX_ROW_PINS
+#ifdef MATRIX_COL_PINS
+static pin_t col_pins[MATRIX_COLS] = MATRIX_COL_PINS;
+#endif // MATRIX_COL_PINS
+
+#define ROWS_PER_HAND MATRIX_ROWS
+
+static inline void setPinOutput_writeLow(pin_t pin) {
```

Comment:
Suggestion to make function names consistent: convert to underscore
-> `set_pin_output_write_low(pint_t pin)`
(same for the rest of pin functions below)

Reply:
Thanks!